better-font-awesome.php

```diff
@@ -453,26 +481,38 @@ public function version_callback( $versions ) {
             ?>
             <p>
                 <?php 
-                printf( __( 'Version selection is currently unavailable. The attempt to reach the jsDelivr API server failed with the following error: %s', 'better-font-awesome' ), 
-                    '<code>' . $this->bfa_lib->get_error('api')->get_error_code() . ': ' . $this->bfa_lib->get_error('api')->get_error_message() . '</code>'
+                printf(
+	                esc_html__(
+		                'Version selection is currently unavailable. The attempt to reach the jsDelivr API server failed with the following error: %s',
+		                'better-font-awesome'
+	                ),
+	                '<code>' . $this->bfa_lib->get_error('api')->get_error_code() . ': ' . $this->bfa_lib->get_error('api')->get_error_message() . '</code>'
```

Comment:
@tcrsavage do I need to `esc_html` this as in `'<code>' . esc_html( $this->bfa_lib->get_error('api')->get_error_code() ) .` ?